packages/commonwealth/client/scripts/views/pages/MyCommunityStake/Transactions/Transactions.tsx

```diff
@@ -57,12 +58,34 @@ const columnInfo = [
   },
 ];
 
-const Transactions = () => {
+type TransactionsProps = {
+  filterOptions: FilterOptions;
```

Comment:
do we need to pass object if there is only `searchText` prop inside?

Reply:
it will be extended later on in https://github.com/hicommonwealth/commonwealth/issues/6677